app/index.js

```diff
@@ -21,7 +21,7 @@ var path = require('path');
 var mkdirp = require('mkdirp');
 const utils = require('./util');
 
-const defaultCamelVersion = "2.18.1";
+const defaultCamelVersion = "2.18.2";
```

Comment:
why are we using this "old" Camel version? What about using latest one (2.22.1)? or latest corresponding to a Fuse version(2.21.0)?